jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -997,6 +997,17 @@ public static void load(String pathName) {
  */
 @CallerSensitive
 public static void loadLibrary(String libName) {
+	if (libName.indexOf(File.pathSeparator) >= 0) {
+		/*[MSG "K0B01", "Library name should not contain a file path: {0}]*/
```

Comment:
I think 'must' would be more appropriate than 'should' in this message.

Reply:
Updated